src/jinja2_fragments/__init__.py

```diff
@@ -85,3 +85,57 @@ def render_block(
         return environment.concat(block_render_func(ctx))  # type: ignore
     except Exception:
         environment.handle_exception()
+
+
+def render_block_list(
```

Comment:
What do you think of naming this `render_blocks` instead? I'm on the camp that naming variables after their type is generally discouraged.

Reply:
I'm all for it, it was the initial name but I changed it to `render_block_list` because I was unsure about both functions having such similar names